handbook/ldap/ldap.md

```diff
@@ -0,0 +1,120 @@
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink.
+    - Refer to [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
+
+- Administrator desktop access to the SystemLink server.
+
+- An LDAP server accessible to the SystemLink server.
+
+- A bind user and bind password for the LDAP server.
+
+- Familiarity with the LDAP attributes available to your organization.
+    - If you do not know what LDAP attributes and groups are available to you, talk with your LDAP system administrator.
+    - Tools such [ADExplorer](https://docs.microsoft.com/en-us/sysinternals/downloads/adexplorer) can be helpful to explore the attributes assigned to your users.
+
+## Enabling LDAP in SystemLink
+
+### Configuring Authentication
+
+1. Log into the server running SystemLink and open **NI Web Sever Configuration**.
+
+2. Go to the **Authentication** tab and enable **Connect to an LDAP server**.
+
+3. Enter the LDAP URL for your sever. Refer to [**LDAP URLs**](#ldap-url) for details on how to structure this URL.
+
+4. Enter the bind user. Refer to [**Bind User**](#bind-user) for details on usernames that can be used here.
+
+5. Enter the bind password.
+
+6. Click **Apply and Restart**.
+
+<figure>
+  <img src="../../img/ldap-setup.png" width="500" />
+  <figcaption>Configuration needed to enable logging into a SystemLink server using LDAP credentials.</figcaption>
+</figure>
+
+You may now login to the SystemLink server using your LDAP credentials. To access systems and data in SystemLink, configure workspace and role mappings. Refer to [**Mapping LDAP Attributes and Groups to SystemLink Workspaces and Roles**](#mapping-ldap-attributes-and-groups-to-systemlink-workspaces-and-roles) for details.
+
+<figure>
+  <img src="../../img/no-mapping.png" width="500" />
+  <figcaption>View from a user who can login but does not have any workspace and role mappings.</figcaption>
+</figure>
+
+## LDAP URL and Bind User
+
+You must provide a URL and bind user and bind password to establish an authenticated connection between your SystemLink server and LDAP server.
+
+### LDAP URL
+
+The LDAP URL follows a standard scheme. The URL determines the server to connect to, the server's port, a base search distinguished name (DN), and the attribute that determines the SystemLink username for login.
+
+!!! note "Basic LDAP URL Scheme"
+
+    **Structure:** `ldap://<server-dns>:<port>/<target-entry-dn>?<username-attribute>`
+
+    **Example:** `ldap://example.com:389/dc=example,dc=com?sAMAccountName`
+
+    In this example the server is `example.com`, the port is 389, the base search DN is `dc=example,dc=com` and the LDAP attribute used for user login is `sAMAccountName`. 
```

Comment:
It may be worth adding a note/troubleshooting somewhere mentioning that when your LDAP server is backed by Windows Active Directory, you may want to point to the global catalog on port 3268 in order to log in cross-domain ("forest" to use the Microsoft terminology).

Related to https://dev.azure.com/ni/DevCentral/_workitems/edit/100255/ where we ran into issues with LDAP referrals (redirects) when one LDAP server (e.g. AMER) wants to redirect to another one (e.g. NI). Using the global catalog avoids the issue, because it has access to every domain in the forest.

Reply:
I added the details about port 3268 in an expanded description of the port in the URL.